modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -69,3 +69,24 @@ $ oc apply -f max-worker-pods.yaml
 ----
 $ oc get kubeletconfigs set-max-pods -o yaml
 ----
+
+By default, only 1 machine is allowed to be unavailable when applying the kubelet-related config to the available worker nodes. This will take a lot of time for the config change to get reflected for a large cluster. The number of machines that can be updating at any given time can be tweaked for speeding up the process.
```

Comment:
s/1/one
s/config/configuration 

For a large cluster, it can take a long time for the configuration change to be reflected.

^^ how long is a long time? seconds? minutes?

At any time, you can adjust the number of machines that are updating to speed up the process.

Reply:
The time depends on the maxUnavailable value we set in the config. We don't have the exact time it took for the change to get applied on the 250 worker nodes as we changed it multiple times to speed up the process. Using the default value ( 1 unavailable machine ), it took around 80 minutes for 100 nodes.